phoenix-core/src/it/java/org/apache/phoenix/end2end/CursorWithRowValueConstructorIT.java

```diff
@@ -669,4 +672,94 @@ public void testCursorsWithColsOfTypesTinyintSmallintFloatDouble() throws Except
             conn.close();
         }
     }
+
+    @Test
+    public void testCursorWithIndex() throws Exception {
+        Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
+        Connection conn = DriverManager.getConnection(getUrl(), props);
+
+        Statement stmt = conn.createStatement();
+
+        stmt.execute("DROP TABLE IF EXISTS SOME_NUMBERS");
```

Comment:
Please use the same table name generation that we use in the other tests.
It reduces the risk accidental table name clashes later, and is a bit faster.